tutorials/misc/plot_report.py

```diff
@@ -203,6 +203,26 @@
 
 report.save('report_custom.html', overwrite=True)
 
+###############################################################################
+# Adding a stc plot to a report
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Now we see how :class:`~mne.Report` handles :class:`~mne.SourceEstimate`
+# data. The following will produce a stc plot with vertex time courses. In
+# this scenario, we also demonstrate how to use the ``screenshot`` function
+# to save the figs in a slider.
+
+fname_stc = os.path.join(path, 'MEG', 'sample', 'sample_audvis-meg')
+stc = mne.read_source_estimate(fname_stc, subject='sample')
+figs = list()
+for hemi in ('lh', 'rh'):
+    brain = stc.plot(subjects_dir=subjects_dir, initial_time=0.13,
+                     clim=dict(kind='value', lims=[3, 6, 9]), hemi=hemi)
+    figs.append(brain.screenshot(time_viewer=True))
```

Comment:
does this save an image to disk? might be a problem if you have many figures and it would pollute your local disk. Part of the beauty of passing a figure object is that you don't have to save anything

Reply:
No, it's all in memory/ndarray

Reply:
So I just gave this a shot. The problem with the current solution is that all the buttons occlude the view. I think you only want the timestamp and colorbar other than the brain

Reply:
```suggestion
    brain.toggle_interface(False)
    figs.append(brain.screenshot(time_viewer=True))
```